### ISSUE TITLE
RqMtFakeTest.java:368-372: The test below fails with the...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/takes#950]
The puzzle `577-dcc6f7a1` from #577 has to be resolved: 

https://github.com/yegor256/takes/blob/d5f747d2586acc328d5a427599bc8eeb6e26253c/src/test/java/org/takes/rq/multipart/RqMtFakeTest.java#L368-L372

The puzzle was created by @rultor on 19-Jan-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#950](https://github.com/yegor256/takes/issues/950) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#950](https://github.com/yegor256/takes/issues/950) assigned to @marceloamadeu[/z](https://www.0crat.com/u/marceloamadeu), here is [why](https://www.0crat.com/footprint/C3FFK3YAY/29bf7b6c-cbcb-43c9-bdcb-348b21058395); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by marceloamadeu** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/takes/issues/950#issuecomment-465735426))

@marceloamadeu The user @marceloamadeu[/z](https://www.0crat.com/u/marceloamadeu) resigned from [#950](https://github.com/yegor256/takes/issues/950), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @marceloamadeu[/z](https://www.0crat.com/u/marceloamadeu)

**Comment by 0crat** [comment]
The job [#950](https://github.com/yegor256/takes/issues/950) assigned to @serranya[/z](https://www.0crat.com/u/serranya), here is [why](https://www.0crat.com/footprint/C3FFK3YAY/3f88705a-1b0e-4447-b868-d25c68136e11); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@844d3d10-572e-4d86-bb4b-e67ee3eddac3, thread: pool-4-thread-3
com.zerocracy.farm.reactive.StkRuntime[76] java.lang.IllegalStateException: org.codehaus.groovy.runtime.InvokerInvocationException in com.zerocracy.stk.pmo.agenda.set_agenda_title_from_github
org.codehaus.groovy.reflection.CachedMethod[105] org.codehaus.groovy.runtime.InvokerInvocationException: java.io.IOException: failed after 4 attempts
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.42.1](https://github.com/zerocracy/farm): CID: [35dd9ed3-3e9d-48ad-ab37-3c077176b198](https://www.0crat.com/footprint/C3FFK3YAY/35dd9ed3-3e9d-48ad-ab37-3c077176b198), Type: "Agenda was updated"

**Comment by Serranya** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/takes/issues/950#issuecomment-474707454))

@Serranya The user @serranya[/z](https://www.0crat.com/u/serranya) resigned from [#950](https://github.com/yegor256/takes/issues/950), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @serranya[/z](https://www.0crat.com/u/serranya)

**Comment by 0crat** [comment]
The job [#950](https://github.com/yegor256/takes/issues/950) assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C3FFK3YAY/097b7c07-8df1-4ca6-958c-77cdbbb16fba); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by paulodamaso** [comment]
@0crat assign @andreoss

**Comment by 0pdd** [comment]
The puzzle `577-dcc6f7a1` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1095](https://github.com/yegor256/takes/issues/1095) is still not solved.

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @andreoss[/z](https://www.0crat.com/u/andreoss), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1131](https://github.com/yegor256/takes/issues/1131) is still not solved.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#unknown]() is still not solved; solved: [#1131](https://github.com/yegor256/takes/issues/1131).

**Comment by 0pdd** [comment]
@0pdd the only puzzle [#1095](https://github.com/yegor256/takes/issues/1095) is solved here.
